program/src/processor/set_authority.rs

```diff
@@ -13,9 +19,85 @@ use crate::instruction::{
 ///   1. `[s]` config_authority
```

Comment:
nit: I can't comment higher up but can you change line 12 to "0. `[w]` config or stake account" and then line 13 to "1. `[s]` current authority"?